walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -121,6 +124,64 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void everyAttendeeIsConsideredOptionalAttendeeUnavailable() {
```

Comment:
Use this sample format to name your tests:[functionName]With[Condition]Returns[What'sExpected]

Reply:
Got it. I'll keep that in mind, but since most of the tests are from the tutorial and I only modified some of them to create 5 or 6 tests, it would be a lot of work to rename all of them in this format. Can I just have a pass this time?